apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1267,7 +1293,19 @@ handle_common_event_(cast, {?DISCONNECT, _}, _St, _, D) ->
 handle_common_event_(cast, {?CHANNEL_CLOSING, ChanId} = Msg, _St, _,
                      #data{on_chain_id = ChanId} = D) ->
     lager:debug("got ~p", [Msg]),
-    close(channel_closing_on_chain, D);
+    D1 = log(rcv, ?CHANNEL_CLOSING, Msg, D),
+    case check_closing_event(D1) of
```

Comment:
The `check_closing_event/1` function is actually buggy, and will (as far as I can tell) always return an error. This happens to result in the old behavior (fsm terminating if it detects a change on-chain). Fixing it in this PR would open a can of worms, since it would require further changes in the watcher and fsm.

One could argue that `check_closing_event/1` should be short-circuited for now rather than accidentally failing in a fortuitous way.